src/app/Http/Controllers/QuizController.php

```diff
@@ -15,7 +15,7 @@ public function list(Request $request)
     {
         // big_questions テーブルのデータを全て取得し $links に格納
         // $links = DB::table('bigquestions')->get(); 
-        $links = BigQuestion::all();
+        $links = BigQuestion::where('hide', 0)->get();
```

Comment:
active, public, openなどのスコープ名で作成した方が明示的かと思います。

```suggestion
        $links = BigQuestion::active()->get();
```